provider/cluster_rosa_classic_resource.go

```diff
@@ -644,6 +812,64 @@ func isByoOidcSet(sts *Sts) bool {
 	return !sts.OIDCEndpointURL.Unknown && !sts.OIDCEndpointURL.Null && sts.OIDCEndpointURL.Value != "" ||
 		!sts.OIDCPrivateKeySecretArn.Unknown && !sts.OIDCPrivateKeySecretArn.Null && sts.OIDCPrivateKeySecretArn.Value != ""
 }
+func (r *ClusterRosaClassicResource) getVersionList(logger logging.Logger, ctx context.Context) (versionList []string, err error) {
+	vs, err := r.getVersions(logger, ctx)
+	if err != nil {
+		err = fmt.Errorf("Failed to retrieve versions: %s", err)
+		return
+	}
+
+	for _, v := range vs {
+		versionList = append(versionList, v.RawID())
+	}
+
+	if len(versionList) == 0 {
+		err = fmt.Errorf("Could not find versions")
+		return
+	}
+
+	return
+}
+func (r *ClusterRosaClassicResource) getVersions(logger logging.Logger, ctx context.Context) (versions []*cmv1.Version, err error) {
```

Comment:
I think this is outdated.
https://github.com/openshift/rosa/blob/master/pkg/ocm/versions.go#L36 - we select now the **latest** as the default.

Reply:
I updated  it 
thanks